bentoml/_internal/io_descriptors/json.py

```diff
@@ -115,17 +110,19 @@ def __init__(
         json_encoder: t.Type[json.JSONEncoder] = DefaultJsonEncoder,
     ):
         if pydantic_model is not None:  # pragma: no cover
```

Comment:
now we have the `LazyType` abstraction, should we do a `is_instance` check here?